omero/developers/testing.txt

```diff
@@ -362,18 +362,27 @@ To view all available markers the :option:`--markers` option can be used with
 :program:`setup.py test` or :program:`py.test` as detailed in
 :ref:`running-python-tests-directly`.
 
-There are currently two custom markers defined:
+There are a small number of custom markers defined:
```

Comment:
Hmm, not sure. I had originally changed it to 'three' and then thought better of it. I then tried 'some' but that seemed a bit vague.